plugin/completion.py

```diff
@@ -41,10 +42,76 @@
 
 
 class LspResolveDocsCommand(sublime_plugin.TextCommand):
+    def run(self, edit: sublime.Edit, index: int) -> None:
+        item = CompletionHandler.completions[index]
+        detail = minihtml(item.get('detail') or "", self.view)
+        documentation = minihtml(item.get("documentation") or "", self.view)
+
+        # don't show the detail in the cooperate AC popup if it is already shown in the AC details filed.
+        self.is_detail_shown = bool(detail)
+
+        content = self.get_content(documentation, detail)
+        self.show_popup(content)
+
+        if not detail or not documentation:
+            # To make sure that the detail or documentation fields doesn't exist we need to resove the completion item.
+            # If those fields appear after the item is resolved we show them in the popup.
+            self.do_resolve(item)
+
+    def get_content(self, documentation: str, detail: str) -> str:
+        content = ""
+        if detail and not self.is_detail_shown:
+            content += """<div class='highlight' style='margin: 4px'>
+                <div>{}</div>
+            </div>""".format(detail)
+
+        if documentation:
+            content += """<div style='margin: 10px'>
+                <div>{}</div>
+            </div>""".format(documentation)
```

Comment:
We don't need extra child `<div>`. And we don't need whitespace in those strings (both can be on one line).

Reply:
ca0fbf72ef62b1fbe312e918d099832fa7f284d0